audit/projects/k8s-infra-prow-build-trusted/secrets/cncf-ci-github-token/iam.json

```diff
@@ -2,6 +2,7 @@
   "bindings": [
     {
       "members": [
+        "group:k8s-infra-ii-coop@kubernetes.io",
```

Comment:
FYI as a followup @hh I'm manually deleting your `user:` binding; it's redundant since you're part of the above group.

Plus, just trying to prune `user:` bindings in general; if it's important enough for a person to have access, it's important enough for a group to have access and hold each other accountable.

```
$ gcloud secrets --project=k8s-infra-prow-build-trusted remove-iam-policy-binding cncf-ci-github-token --member=user:hh@ii.coop --role=roles/secretmanager.admin
Updated IAM policy for secret [cncf-ci-github-token].
bindings:
- members:
  - group:k8s-infra-ii-coop@kubernetes.io
  - group:k8s-infra-prow-oncall@kubernetes.io
  role: roles/secretmanager.admin
etag: BwXDGOENV5g=
version: 1
```